src/textual/widget.py

```diff
@@ -618,6 +618,33 @@ def mount(
         self.call_next(await_mount)
         return await_mount
 
+    def mount_all(
+        self,
+        widgets: Iterable[Widget],
+        *,
+        before: int | str | Widget | None = None,
+        after: int | str | Widget | None = None,
+    ) -> AwaitMount:
+        """Mount widgets from an iterable.
+
+        Args:
+            widgets: An iterable of widgets.
+            before: Optional location to mount before.
+            after: Optional location to mount after.
```

Comment:
Wee reminder for when this goes non-draft, yesterday I expanded the documentation for `before`/`after` so this may want to follow that pattern: https://github.com/Textualize/textual/blob/2662e6b2923d0f5aad8abff1281d463036e7d3d0/src/textual/widget.py#L564-L569

Also you've reminded me that I *didn't* update `App.mount_all` with this (I'll do that).